container-runtime/src/Adapters/RpcListener.ts

```diff
@@ -247,7 +247,7 @@ export class RpcListener {
           })),
       )
       .when(exitType, async ({ id }) => {
-        if (this._system) await this._system.exit(this.effects)
+        if (this._system) await this._system.exit(this.effects("exitTypeId"))
```

Comment:
Use `null` and make it nullable. OS will accept null if it gets passed

Reply:
Thanks